src/app/(main)/my-voyage/[teamId]/sprints/sprintsService.ts

```diff
@@ -61,18 +61,19 @@ type EditSectionType = Pick<
 
 export interface SprintsResponse {
   id: number;
-  name: string;
-  voyage: {
+  number: string;
```

Comment:
I think it's a little confusing having this be named number since we have number in the entire sprints object (and there's no voyage anymore) and then also number in the sprints array as well. I think it'd be better to name this voyage_number.

Reply:
Hmmm, but it's what we get from the backend... not sure how I can fix it 😓

Reply:
In your fetch sprints function you can manipulate the data and return it and then that will dispatch the transformed data to the redux store and will be used throughout the app, similar to what I did in directory page.

Also, I think I meant to change this naming in the redux store, not here lol. It's not possible to change it here.

Reply:
Sure you want me to change it now? a `voyage` object is back in `sprintSlice`😅

Reply:
I think your update is good and solves this